api/v1alpha1/worker_types.go

```diff
@@ -273,6 +274,23 @@ const (
 
 type GateWorkflowConfig struct {
 	WorkflowType string `json:"workflowType"`
+	// Input is an arbitrary JSON object passed as the first parameter to the gate workflow.
```

Comment:
can you add to this comment that the input will be logged, but that InputFrom will not be logged?